src/gatsby/plugins/gatsby-plugin-openapi/gatsby-node.ts

```diff
@@ -32,6 +32,36 @@ export const sourceNodes = async (
 
     const parsedContent = parseContent();
 
+    // temporarily create a mock api description so the apiDoc query doesnt fail
```

Comment:
seems weird but this is what gatsby recommends doing

Reply:
@JoshFerge hmm why is there no nodes present. I see `'x-display-description'` exists in the derefed openapi. https://github.com/getsentry/sentry-api-schema/blob/main/openapi-derefed.json#L25

Reply:
kk went ahead and removed this -- will wait to merge until other scim PRs are in.